### PR TITLE
Add check for ages > 90. Closes #208

### LINKED ISSUE #208: Add check for ages over 90
In human individual data we should not have ages over 90. These should be represented as the string "90+". We could add a check for ages over 90. Marking as low priority because writing a custom function for this may not be necessary if we switch to JSON Schema.

### PR DESCRIPTION
**Pull request by karawoo**
Fixes #208

Changes proposed in this pull request:

- Adds a new check function, `check_ages_over_90()`, to check for uncensored ages >90. `check_all()` will run this check for human data only.

Please confirm you've done the following (if applicable):

- Added tests for new functions or for new behavior in existing functions: yes
- If adding a new exported function, added it to the reference section of `_pkgdown.yml`: yes
- If adding a new configuration option, documented it in `vignettes/customizing_dccvalidator.Rmd`: NA

### CODE REVIEW

File: R/check-all.R [hunk 1]
```diff
@@ -240,6 +240,13 @@ check_all <- function(data, annotations, syn) {
   # Parent column in manifest is valid synID -----------------------------------
   valid_parent_syn <- check_parent_syn(data$file_data[manifest_index][[1]])
 
+  # Ages over 90 are censored in human individual metadata ---------------------
+  if (all(data$species == "human")) {
```
[the remark this answers is not in the document]
Comment (karawoo) [reply R/check-all.R:244]: Oh yeah, good to know, I'll switch it to `any()`

**Comment by karawoo** [comment]
Huh, I could have sworn it was working when I was testing earlier but now I'm now seeing a failure locally as well. `check_all()` doesn't seem to like `"mouse or other animal model"` as a species -- @Aryllen do you know why that might be?

```
✖ |  11 1     | test-check-all.R [6.5 s]
────────────────────────────────────────────────────────────────────────────────
test-check-all.R:224: error: check_all runs check_ages_over_90 for human data
Couldn't download metadata template. Make sure you are logged in to Synapse and that `synID` is a valid synapse ID.FALSE
1: check_all(data_animal, annots, syn) at tests/testthat/test-check-all.R:224
2: check_cols_individual(data$file_data[indiv_index][[1]], config::get("templates")$individual_templates[[data$species[indiv_index]]], 
       syn = syn) at /Users/kwoo/projects/validation/dccvalidator/R/check-all.R:66
3: get_template(id, ...) at /Users/kwoo/projects/validation/dccvalidator/R/check-col-names.R:78
4: stop("Couldn't download metadata template. Make sure you are logged in to Synapse and that `synID` is a valid synapse ID.", 
       .call = FALSE) at /Users/kwoo/projects/validation/dccvalidator/R/check-col-names.R:160
```

**Comment by karawoo** [comment]
Never mind, I think I found it

**Comment by Aryllen** [comment]
That was odd because it worked for me locally when I tested it, as well. It shouldn't have. XD

Thank you for finding and fixing the problem.

**Comment by karawoo** [comment]
I think it depends on the working directory -- my previous R session might have been looking at the main app config instead of the test config?

**Pull request merged**